### ISSUE TITLE
Space + Mouse no longer moves canvas.

### ISSUE THREAD

**Issue by cor-of-org** [comfyanonymous/ComfyUI#5397]
### Expected Behavior

Pressing the spacebar and moving the mouse moves the canvas.

### Actual Behavior

Pressing Spacebar now changes the icon on a pointless toolbar which has appeared, without permission, in the bottom-right of my viewport.

There is nothing useful in this toolbar, except letting me know that yes, I am pressing the SPACEBAR now, and moving my mouse and yes, nothing is ******* happening.

### Steps to Reproduce

See above.

Basically, press down the spacebar on your keyboard and move the mouse around. Notice how nothing is happening?

That used to be a super-useful combo that folk like me used hundreds of times a day to save time but now makes them want to punch things

### Debug Logs

```powershell
N/A.
```

**Comment by huchenlei** [comment]
Now press space enters the pan mode, which locks the canvas for panning. You still need to hold down left mouse button to do the actual panning. This aligns with most panning mode design in other design softwares like Photoshop.
